frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
You're right. I didn't notice until now. It happens only because of my removal of `JSON.parse(JSON.stringify())` before passing the geojson to `geojsonExtent`. If I readd it, everything works fine.